framework/src/mysql/mysql.js

```diff
@@ -414,113 +424,52 @@ const getTableInstance = async (tableName, tableConfig, connEndpoint = CONN_ENDP
-				const { property, values } = param;
-				query.whereIn(property, values);
-			});
-		}
-
-		if (params.whereJsonSupersetOf) {
-			const { property, values } = params.whereJsonSupersetOf;
-			query.where(function () {
-				const [val0, ...remValues] = Array.isArray(values) ? values : [values];
-				this.whereJsonSupersetOf(property, [val0]);
-				remValues.forEach(value => this.orWhere(function () {
-					this.whereJsonSupersetOf(property, [value]);
-				}));
-			});
+	const find = async (params = {}, columns, trx) => {
+		let isDefaultTrx = false;
+		if (!trx) {
+			trx = await createDefaultTransaction(knex);
+			isDefaultTrx = true;
 		}
 
-		if (params.andWhere) {
-			const { andWhere } = params;
-			query.where(function () {
-				this.where(andWhere);
-			});
+		if (!columns) {
+			logger.warn(`No SELECT columns specified in the query, returning the '${tableName}' table primary key: '${tableConfig.primaryKey}'`);
+			columns = Array.isArray(tableConfig.primaryKey)
+				? tableConfig.primaryKey : [tableConfig.primaryKey];
 		}
+		const query = queryBuilder(params, columns, trx);
+		const debugSql = query.toSQL().toNative();
+		logger.debug(`${debugSql.sql}; bindings: ${debugSql.bindings}`);
 
-		if (params.orWhere) {
-			const { orWhere, orWhereWith } = params;
-			query.where(function () {
-				this.where(orWhere).orWhere(orWhereWith);
+		if (isDefaultTrx) return query
+			.then(async response => {
+				await trx.commit();
+				return response;
+			}).catch(async err => {
+				await trx.rollback();
+				logger.error(err.message);
+				throw err;
 			});
-		}
 
-		if (params.orWhereIn) {
-			const { property, values } = params.orWhereIn;
-			query.orWhereIn(property, values);
-		}
+		return query;
+	};
 
-		if (params.search) {
-			params.search = Array.isArray(params.search) ? params.search : [params.search];
+	const count = async (params = {}, column) => {
+		const trx = await createDefaultTransaction(knex);
+		params.count = true;
 
-			params.search.forEach(search => {
-				const { property, pattern, startsWith, endsWith } = search;
-				if (pattern) query.where(`${property}`, 'like', `%${pattern}%`);
-				if (startsWith) query.where(`${property}`, 'like', `${startsWith}%`);
-				if (endsWith) query.where(`${property}`, 'like', `%${endsWith}`);
-			});
+		if (!column) {
+			logger.warn(`No SELECT columns specified in the query, returning the '${tableName}' table primary key: '${tableConfig.primaryKey}'`);
+			column = Array.isArray(tableConfig.primaryKey)
+				? [tableConfig.primaryKey[0]] : [tableConfig.primaryKey];
+		} else {
+			column = Array.isArray(column) ? [column[0]] : [column];
 		}
 
-		if (params.orSearch) {
-			params.orSearch = Array.isArray(params.orSearch) ? params.orSearch : [params.orSearch];
-
-			query.andWhere(function () {
-				params.orSearch.forEach((orSearch, index) => {
-					const { property, pattern, startsWith, endsWith } = orSearch;
+		const query = queryBuilder(params, column, trx);
```

Comment:
Remove query.count on L458. Please make the same changes in sqlite.js as well.
```suggestion
		const query = queryBuilder({ ...params, count: true }, column, trx);
```